src/index.js

```diff
@@ -26,9 +26,9 @@ function crossEnv(args) {
 
 function getCommandArgsAndEnvVars(args) {
   const envVars = getEnvVars()
-  const commandArgs = args.map(commandConvert)
-  const command = getCommand(commandArgs, envVars)
-  return [command, commandArgs, envVars]
+  const commandArgs = args.slice()
+  const command = commandConvert(getCommand(commandArgs, envVars))
+  return [commandConvert(command), commandArgs.map(commandConvert), envVars]
```

Comment:
> You are leveraging the fact that getCommand will strip env setters and the command from commandArgs. And you are making a copy so that getCommandArgsAndEnvVars does not itself mutate its argument. Am I correct?

Correct. If I didn't make a copy, a call to `crossEnv` would get its arguments mutated, and I don't think we want that :)

> A possible improvement would be to replace it with 2 functions...

That sound good, this part was the most confusing when I started looking at this repo, so I think it's totally worth it to refactor a bit.